packages/numpy/test_numpy.py

```diff
@@ -107,11 +107,6 @@ def assert_equal():
 
     assert selenium.run("np.array([True, False])") == [True, False]
 
-    selenium.run("x = np.array([['string1', 'string2'], ['string3', 'string4']])")
```

Comment:
What's the situation with numpy arrays containing strings after this PR?

Reply:
I'm confused. I must have thought it was correct for this test to fail because I deleted it, but I don't remember why.
The behavior is really weird and inconsistent though, definitely should be improved. I'm not sure what it should do though. Some examples:

```javascript
ajs = pyodide.runPython(`
   a = np.array([0, 1])
   a
`).toJs()
```
`ajs` is an `Int32Array` pointing to the same memory as `a` and suffering from #1221.

```javacript
bjs = pyodide.runPython(`
   b = np.array([True, False])
   b
`).toJs()
```
`bjs` is an Array containing `[true, false]`! Even though `b[0]` is of type `numpy.bool_`. `b[0]` actually makes a `PyProxy` such that when you call `toJs()` on it it turns to `true`. No idea how that happens, I will probably have to insert some print statements into the conversion code and trace through what the heck is going on.

Lastly 
```javacript
cjs = pyodide.runPython(`
   c = np.array(["abc", "def"])
   c
`).toJs()
```
is still a proxy.

Reply:
OK, let's address this in a separate issue.